modules/ROOT/pages/logger-component-reference.adoc

```diff
@@ -6,74 +6,99 @@ endif::[]
 
 This Core component helps you monitor and debug your Mule application by logging important information such as error messages, status notifications, payloads, and so on. You can add a Logger anywhere in a flow, and you can configure it to log a string that you specify, the output of a DataWeave expression you write, or any combination of strings and expressions.
 
-[WARNING]
---
-Keep in mind that the Logger is one of the only components that supports mixing Strings and expressions within a value. DataWeave String interpolation or concatenation within a single expression should be used elsewhere.
---
+//TODO: IS THIS TRUE BEYOND ON-PREM INSTALLATIONS? 
+The configured messages are logged in a Mule application's log file, which is located in `MULE_HOME/logs/<app-name>.log` if no custom log file path is specified in the `log4j2.xml` file. For details, see xref:logging-and-debugging.adoc[].
 
-The configured messages are logged to the app's log file, which is located in `MULE_HOME/logs/<app-name>.log` if no custom log file path is specified in the `log4j2.xml` file. +
-In Studio, the logs show in the Console.
+In Anypoint Studio, the logs print to the xref:studio::index.adoc#console[console].
 
-The following example displays the message in Set Payload in a browser and also logs the message.
```

Comment:
it's in the browser bc of the request to the listener, not directly relevant to logger